Riot/Modules/Home/AllChats/AllChatsViewController.swift

```diff
@@ -985,8 +985,22 @@ extension AllChatsViewController: SplitViewMasterViewControllerProtocol {
     private func presentVerifyCurrentSessionAlert(with session: MXSession) {
         MXLog.debug("[AllChatsViewController] presentVerifyCurrentSessionAlertWithSession")
         
-        let alert = UIAlertController(title: VectorL10n.keyVerificationSelfVerifyCurrentSessionAlertTitle,
-                                      message: VectorL10n.keyVerificationSelfVerifyCurrentSessionAlertMessage,
+        let title: String
+        let message: String
+        
+        if
+            let feature = MXSDKOptions.sharedInstance().cryptoSDKFeature,
+            feature.isEnabled && feature.needsVerificationUpgrade
+        {
```

Comment:
I think this would be more inline with the project's style:
```suggestion
        if let feature = MXSDKOptions.sharedInstance().cryptoSDKFeature,
           feature.isEnabled && feature.needsVerificationUpgrade {
```